test/stress/generate-report/generate_report.py

```
@@ -0,0 +1,860 @@
+            WHERE d.delta >= 0
+        )
+        SELECT 
+            phase_name,
+            operation_type,
+            SUM(CASE WHEN metric = 'kubelet_runtime_operations_duration_seconds_count' THEN delta ELSE 0 END) as count_delta,
+            SUM(CASE WHEN metric = 'kubelet_runtime_operations_duration_seconds_sum' THEN delta ELSE 0 END) as sum_delta
+        FROM diffs_with_phase
+        GROUP BY phase_name, operation_type
+        HAVING count_delta > 0
+        ORDER BY phase_name, count_delta DESC
+    """).fetchall()
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)